configs/nextflow.hopper.config

```diff
@@ -113,6 +113,11 @@ profiles {
     params.gatkcnv = true
     params.umi = false
     params.delly = false
+    params.run_melt = false
```

Comment:
This is a suggestion on how to avoid the somewhat convoluted profile checks. Let the profiles themselves decide what to run.